.github/workflows/render-readme-evaluation.yml

```
@@ -0,0 +1,42 @@
+on:
+  schedule:
+    - cron: "0 13 * * 2,3"
+  push:
+    paths:
+      - code/evaluation/README.Rmd
+
+name: Render README
+
+jobs:
+  render:
+    if: github.repository_owner == 'covid19-forecast-hub-europe'
+    name: Render README
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: r-lib/actions/setup-pandoc@v1
+
+      - uses: r-lib/actions/setup-r@v2
+        with:
+          install-r: false
+          use-public-rspm: true
+
+      - name: Install system dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install libudunits2-dev libcurl4-openssl-dev libgdal-dev
+
+      - uses: r-lib/actions/setup-renv@v2
+
+      - name: Render README
+        run: Rscript -e 'rmarkdown::render("code/evaluation/README.Rmd")'
+
+      - name: Commit results
+        run: |
+          git config --local user.email "actions@github.com"
+          git config --local user.name "GitHub Actions"
+          git commit README.md -m 'Re-build evaluation README.Rmd' || echo "No changes to commit"
```

Does this have to be:

```suggestion
          git commit evaluation/README.md -m 'Re-build evaluation README.Rmd' || echo "No changes to commit"
```